src/app/api/route.ts

```diff
@@ -0,0 +1,14 @@
+import { callOpenai } from '@/utils/openai';
+import { NextResponse } from 'next/server';
+import dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
```

Comment:
isn't this `../../../`? need to go up three directories to hit `.env.local`

Reply:
This is actually unused, so I removed this!